pkg/gui/diffing.go

```diff
@@ -159,3 +160,35 @@ func (gui *Gui) handleCreateDiffingMenuPanel() error {
 
 	return gui.c.Menu(types.CreateMenuOptions{Title: gui.c.Tr.DiffingMenuTitle, Items: menuItems})
 }
+
+func (gui *Gui) handleLaunchDiffTool() error {
+	gui.c.LogAction("Launch DiffTool")
+
+	cmdStr := "git difftool --submodule "
+
+	file := gui.currentlySelectedFilename()
```

Comment:
```suggestion
	file := gui.currentlySelectedFilename()
	if file == "" {
		file = gui.getSelectedPath()
	}

```

for Staged/Unstaged Changes panel